From e0d12217fe5260fb54e24c02a15cdae1fe3cdf70 Mon Sep 17 00:00:00 2001
From: Pauli <pauli@openssl.org>
Date: Wed, 2 Aug 2023 11:36:46 +1000
Subject: [PATCH 265/381] test: skip FIPS config auto loading based tests if
 feature is disabled

Reviewed-by: Matt Caswell <matt@openssl.org>
Reviewed-by: Shane Lontis <shane.lontis@oracle.com>
(Merged from https://github.com/openssl/openssl/pull/21621)

(cherry picked from commit 9a255aa2bda07286d5516a77b269178fc03779b4)
---
 test/recipes/30-test_defltfips.t | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/test/recipes/30-test_defltfips.t b/test/recipes/30-test_defltfips.t
index 426bd660d1..103f28da76 100644
--- a/test/recipes/30-test_defltfips.t
+++ b/test/recipes/30-test_defltfips.t
@@ -21,6 +21,9 @@ BEGIN {
 use lib srctop_dir('Configurations');
 use lib bldtop_dir('.');
 
+plan skip_all => "Configuration loading is turned off"
+    if disabled("autoload-config");
+
 my $no_fips = disabled('fips') || ($ENV{NO_FIPS} // 0);
 
 plan tests =>
-- 
2.34.1

